internal/config/config.go

```diff
@@ -63,8 +54,12 @@ type GeneratorConfig struct {
 // TypeConfig is the information about which types to render and any data specific to handling of the type.
 type TypeConfig struct {
 	Name string `yaml:"name"`
-	// TypeOverride is the Golang type to override whatever the default detected type would be.
-	TypeOverride string `yaml:"type_override,omitempty"`
+	// FieldTypeOverride is the Golang type to override whatever the default detected type would be for a given field.
+	FieldTypeOverride string `yaml:"field_type_override,omitempty"`
+	// CreateAs is used when creating a new scalar type to determine which Go type to use.
+	CreateAs string `yaml:"create_as,omitempty"`
+	// SkipTypeCreate allows the user to skip creating a Scalar type.
+	SkipTypeCreate bool `yaml:"skip_type_create,omitempty"`
```

Comment:
Shouldn't this allow to skip creating any type?

Reply:
I don't think so in this case.  The code change is easy, but if you include a type in the config, and it relies on another struct, should we be able to skip that dependency and result in broken code?  It makes the most sense in the scalar since they don't have fields was my take.

Reply:
yeah, unless you define that struct in another file, which is arguably an edge case.